Marlin/ultralcd.cpp

```diff
@@ -684,11 +684,13 @@ void kill_screen(const char* lcd_msg) {
 
   #if ENABLED(MENU_ITEM_CASE_LIGHT)
 
-    extern bool case_light_on;
+    extern uint8_t case_light_brightness;  
+    bool case_light_on = (case_light_brightness >= 128) ? true : false;
```

Comment:
Since `setup` initializes `case_light_on` by calling `update_case_light` this should be left uninitialized here.
```cpp
bool case_light_on;
```
Note also, that if a ternary ends with `true : false` you can drop the ternary and just use:
```cpp
 bool case_light_on = case_light_brightness >= 128;
```